components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/api/APIGateway.java

```diff
@@ -18,6 +18,7 @@
 */
```

Comment:
We will require to improve the debug logs. Can we add more debug logs for troubleshooting purposes?